hpaction/tests/test_build_hpactionvars.py

```diff
@@ -135,7 +135,7 @@ def test_user_to_hpactionvars_populates_issues(db):
     assert first.conditions_complained_of_te == "Mold on walls"
 
     assert second.area_complained_of_mc == hp.AreaComplainedOfMC.PUBLIC_AREA
-    assert second.which_room_mc.value == "Public areas"  # type: ignore
+    assert second.which_room_mc.value == "Building-wide"  # type: ignore
```

Comment:
Hey @toolness! Noticed that me changing the names of the issue areas broke this HPA back-end test. I fixed it here but just wanted to make sure there weren't any other HP-related consequences of this copy change. Lmk!

Reply:
Ah right, there shouldn't be any, since outside of tests, all string comparisons are done against the "code" (e.g. `PUBLIC_AREAS") rather than the user-facing string!  NOICE